runtime/onert/backend/cpu/ops/BinaryArithmeticLayer.cc

```diff
@@ -34,20 +34,21 @@ template <nnfw::cker::BinaryArithmeticOpType arithmetic_type, typename T>
 void eval(const IPortableTensor *lhs, const IPortableTensor *rhs, IPortableTensor *output,
           nnfw::cker::BinaryArithmeticOpParam op_params)
 {
-  const bool need_broadcast =
-      nnfw::cker::ProcessBroadcastShapes(getTensorShape(lhs), getTensorShape(rhs), &op_params);
+  const auto lhsShape = getTensorShape(lhs);
+  const auto rhsShape = getTensorShape(rhs);
```

Comment:
@krayzemli Sorry about the late response, but there is a nit-pick - please update the variable name to snake case.

Reply:
@watert Where can I find project codestyle guidelines?

Reply:
I made PR #4566 about that

Reply:
> @watert Where can I find project codestyle guidelines?

Sorry, we don't have one yet. We may have some discussion in our old repo.